README.md

```diff
@@ -100,7 +120,9 @@ Given a document like
 </main>
 ```
 
-If we called `document.getElementById('app').querySelectorAll('ul.ranked-list li')`, we'd get back a NodeList of `<li>1</li>, <li>2</li>, <li>10</li>, <li>11</li>`.
+If we called `document.getElementById('app').querySelectorAll('ul.ranked-list
+li')`, we'd get back a NodeList of `<li>1</li>, <li>2</li>, <li>10</li>,
```

Comment:
Maybe we need to be more precise here. We're not getting back a NodeList of these strings, but rather the _Nodes_ corresponding to the elements.